src/db/models/User.js

```diff
@@ -61,9 +73,8 @@ module.exports = (sequelize, DataTypes) => {
       },
     },
   );
-  User.associate = model => {
+  User.associate = () => {
     // associations can be defined here
-    User.hasMany(model.Article);
```

Comment:
lets bring back this relationship

Reply:
I have added it now. I got it to work by adding the foreign key